### PR TITLE
[Cong Le] Complete Add Competency and Add Question for Competency

### PR DESCRIPTION
**Pull request by ltc2310**

### CODE REVIEW

File: src/Competencies.js [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+  }
+
+  handleSaveActivated = (e) => {
+    let dataUpdate = {
+      "activated": this.state.optionActivated,
+      "name": this.state.competencyName
+    }
+
+    if (_.isEqual(this.state.selectOption, 'Kms_core')) {
+      _.update(this.state.competenciesKMSCore[this.state.keyUpdate], 'activated', () => { return this.state.optionActivated })
```
Comment (hunterbmt) [inline src/Competencies.js]: Arrow function doesn't need return. We can go with () =? this.state.optionActivated

File: src/Competencies.js [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+  }
+
+  handleCancel = (e) => {
+    this.setState({
+      visible: false,
+    });
+  }
+
+  getColumnKMSCore() {
+    let columnsKMSCore = [{
```
Comment (hunterbmt) [inline src/Competencies.js]: Const

File: src/CompetencyRadar.js [hunk 3]
```diff
@@ -34,19 +34,17 @@ const colorList = _([
...
 const getBorderColor = (index) => colorList[index].setAlpha(1).toRgbString();
 
-
-const CompentencyRadar = ({width = 600, height = 'auto', data, competencies}) => {
-  const radarData = _.map(data, (profile, index) => ({
-    label: profile.name,
-    data: _.map(competencies, (competency) => _.isNumber(profile.competencies[competency]) ?  profile.competencies[competency] : _.find(profile.competencies, {name: competency}).proficiency),
+const CompentencyRadar = ({width = 600, height = 'auto', data, competencies}) => {   
+  const radarData = _.map(data, (item, index) => ({
+    label: 'Must change',
```
Comment (hunterbmt) [inline src/CompetencyRadar.js]: Cannot have the same label

File: src/GroupManagement.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  }
+
+  getGroupDataToState = (projectName) =>  {
+    this.setState({
+      loading: true
+    })
+    getData(`BU_projects/${projectName}`)
+    .then(({members, manager}) => this.setState({
+      projectName: projectName,
+      members: members,
```
Comment (hunterbmt) [inline src/GroupManagement.js:51]: please use shorthand

File: src/PersonalInformation.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  constructor(props) {
+    super(props)
+    this.state = {
+      loading: true,
+      managerInformation: []
+    }
+  }
+
+  findManagerAndTeam = (members, id) => {
+    let result = false
```
Comment (hunterbmt) [inline src/PersonalInformation.js]: Please replace with _.some

File: src/PersonalInformation.js [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    _.forEach(members, (value) => {
+      if (_.isEqual(_.toNumber(id), value)) {
+        result = true
+      }
+    })
+    return result
+  }
+
+  findManagerInformation = (allProjects, id) => {
+    let manager = _.find(allProjects, (project) => {
```
Comment (hunterbmt) [inline src/PersonalInformation.js]: please use const

**Review by hunterbmt — CHANGES_REQUESTED** [review]

File: src/CompetencyRadar.js (hunk 3 above)
Comment (ltc2310) [reply src/CompetencyRadar.js]: Hiep Tran is assigned to fix it

File: src/GroupManagement.js (hunk 4 above)
Comment (ltc2310) [reply src/GroupManagement.js:51]: It's in progress

**Pull request closed without merging**